src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -167,12 +167,14 @@ class FieldBlocks extends React.Component<FieldTypeProps<Array<BlockEntry>>> {
     // eslint-disable-next-line no-unused-vars
     renderCollapsedBlockContent = (value: Object, type: string, index: number) => {
         const blockSchemaType = this.getBlockSchemaType(type);
-        const blockSchemaTypeForm = blockSchemaType.form;
+        const blockSchemaTypeForm = this.removeSections(blockSchemaType.form);
 
         const previewPropertyNames = Object.keys(blockSchemaTypeForm)
             .filter((schemaKey) => {
                 const schemaEntryTags = blockSchemaTypeForm[schemaKey].tags;
-                return schemaEntryTags && schemaEntryTags.some((tag) => tag.name === BLOCK_PREVIEW_TAG);
+                return schemaEntryTags &&
+                    value[schemaKey] &&
```

Comment:
Why is that necessary now?

Reply:
It is not necessary for the block preview.

Currently the first fields are rendered no matter if these fields have values in it or not, with this change the first fields with a value are rendered.